<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTriangleWidget</class>
 <widget class="QWidget" name="EditTriangleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>248</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="3">
       <widget class="QGroupBox" name="m_centerOnGroup">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="title">
         <string>center on</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="m_originButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>origin</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="m_triangleButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>triangle</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="m_nodeXButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="m_markButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>mark</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="m_nodeOButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>O</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="m_nodeYButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="text">
         <string>rotate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_rotateLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="SnapSlider" name="m_rotateSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="minimum">
         <number>-1080</number>
        </property>
        <property name="maximum">
         <number>1080</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="text">
         <string>scale</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="m_scaleLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="SnapSlider" name="m_scaleSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="text">
         <string>move</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <layout class="QHBoxLayout" stretch="0,0,0,0,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="m_moveLeftButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>move left</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
          <property name="icon">
           <iconset resource="../qosmic.qrc">
            <normaloff>:/icons/silk/bullet_arrow_left.xpm</normaloff>:/icons/silk/bullet_arrow_left.xpm</iconset>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_moveRightButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>move right</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
          <property name="icon">
           <iconset resource="../qosmic.qrc">
            <normaloff>:/icons/silk/bullet_arrow_right.xpm</normaloff>:/icons/silk/bullet_arrow_right.xpm</iconset>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_moveDownButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>move down</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qosmic.qrc">
            <normaloff>:/icons/silk/bullet_arrow_down.xpm</normaloff>:/icons/silk/bullet_arrow_down.xpm</iconset>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_moveUpButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>move up</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qosmic.qrc">
            <normaloff>:/icons/silk/bullet_arrow_up.xpm</normaloff>:/icons/silk/bullet_arrow_up.xpm</iconset>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DoubleValueEditor" name="m_multiplierEditor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="text">
         <string>flip</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="m_hFlipButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>flip horizontally</string>
          </property>
          <property name="text">
           <string>horiz</string>
          </property>
          <property name="icon">
           <iconset resource="../qosmic.qrc">
            <normaloff>:/icons/silk/shape_flip_horizontal.xpm</normaloff>:/icons/silk/shape_flip_horizontal.xpm</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_vFlipButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>flip vertically</string>
          </property>
          <property name="text">
           <string>vert</string>
          </property>
          <property name="icon">
           <iconset resource="../qosmic.qrc">
            <normaloff>:/icons/silk/shape_flip_vertical.xpm</normaloff>:/icons/silk/shape_flip_vertical.xpm</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QGroupBox" name="m_modeGroupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="title">
         <string>Selection</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QRadioButton" name="m_selectTrianglesButton">
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>triangles</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="autoRaise" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QRadioButton" name="m_selectNodesButton">
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>nodes</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="autoRaise" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleValueEditor</class>
   <extends>QDoubleSpinBox</extends>
   <header>doublevalueeditor.h</header>
  </customwidget>
  <customwidget>
   <class>SnapSlider</class>
   <extends>QSlider</extends>
   <header>snapslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qosmic.qrc"/>
 </resources>
 <connections/>
</ui>
